data/releases/eol.yaml

```diff
@@ -0,0 +1,66 @@
+branches:
+  - release: 1.21
+    finalPatchRelease: 1.21.14
+    endOfLifeDate: 2022-06-28
+  - release: "1.20"
+    finalPatchRelease: 1.20.15
+    endOfLifeDate: 2022-02-28
+  - release: 1.19
+    finalPatchRelease: 1.19.16
+    endOfLifeDate: 2021-10-28
+  - release: 1.18
+    finalPatchRelease: 1.18.20
+    endOfLifeDate: 2021-06-18
+    note: Created to solve regression introduced in 1.18.19
+  - release: 1.18
+    finalPatchRelease: 1.18.19
+    endOfLifeDate: 2021-05-12
+    note: "Regression: https://groups.google.com/g/kubernetes-dev/c/KuF8s2zueFs"
```

Comment:
This could be a Markdown note:
```markdown
    note: >-
     [Regression](https://groups.google.com/g/kubernetes-dev/c/KuF8s2zueFs)
```

provided that the shortcode supports it.

Reply:
Thanks! This is exactly what I was looking for!